plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp

```diff
@@ -10,16 +10,40 @@
 #include <fc/io/json.hpp>
 #include <fc/exception/exception.hpp>
 #include <fc/reflect/variant.hpp>
+#include <fc/io/json.hpp>
 
 #include <boost/asio/high_resolution_timer.hpp>
 #include <boost/algorithm/clamp.hpp>
 
+#include <Inline/BasicTypes.h>
+#include <IR/Module.h>
+#include <IR/Validate.h>
+#include <WAST/WAST.h>
+#include <WASM/WASM.h>
+#include <Runtime/Runtime.h>
+
+#include <currency/currency.wast.hpp>
+#include <currency/currency.abi.hpp>
+
 namespace eosio { namespace detail {
   struct txn_test_gen_empty {};
 }}
 
 FC_REFLECT(eosio::detail::txn_test_gen_empty, );
 
+static std::vector<uint8_t> assemble_wast( const std::string& wast ) {
+   IR::Module module;
+   std::vector<WAST::Error> parseErrors;
+   WAST::parseModule(wast.c_str(),wast.size(),module,parseErrors);
+   if(parseErrors.size())
+      FC_THROW_EXCEPTION(fc::parse_error_exception, "wast parsing failure");
+
+   // Serialize the WebAssembly module.
+   Serialization::ArrayOutputStream stream;
+   WASM::serialize(stream,module);
+   return stream.getBytes();
+}
+
```

Comment:
I think you can just use `chain::wast_to_wasm()` in `chain/wast_to_wasm.?pp`

Reply:
Good call.